pom.xml

```diff
@@ -12,9 +12,7 @@
 
   <!-- Source Control Management -->
   <scm>
-    <connection>scm:svn:http://anonsvn.jboss.org/repos/jbossws/projects/jaxws-undertow-httpspi/trunk</connection>
-    <developerConnection>scm:svn:https://svn.jboss.org/repos/jbossws/projects/jaxws-undertow-httpspi/trunk</developerConnection>
-    <url>http://fisheye.jboss.com/viewrep/JBossWS/projects/jaxws-undertow-httpspi/trunk</url>
+    <connection>https://github.com/rsearls/jaxws-undertow-httpspi.git</connection>
```

Comment:
you need to retain all 3 elements in <scm>, otherwise the rules that are ran when deploying the artifact to maven repo will likely fail; besides that, a reference to jbossws repo should be used here, not to your personal fork; for reference, this is what we have in RESTEasy:

    <scm>
        <connection>scm:git:git://github.com/resteasy/Resteasy.git</connection>
        <developerConnection>scm:git:git@github.com:resteasy/Resteasy.git</developerConnection>
        <url>https://github.com/resteasy/Resteasy/tree/master/</url>
    </scm>

Reply:
done

Reply:
ok... but this is not RESTEasy ;-) Please fix the URLs

Reply:
fixed

Reply:
@rsearls :

<scm>
    <connection>scm:git:git://github.com/jbossws/jaxws-undertow-httpspi.git</connection>
    <developerConnection>scm:git:git@github.com:jbossws/jaxws-undertow-httpspi.git</developerConnection>
    <url>https://github.com/jbossws/jaxws-undertow-httpspi/tree/master/</url>
</scm>

Reply:
done